float64 posex_from_client
float64 posey_from_client
float64 quaternionz_from_client
float64 quaternionw_from_client
---
float64 posex
float64 posey
float64 quaternionz
float64 quaternionw